Bug ui fixes 163537823

kenzdozz opened the pull request:
#### What does this PR do?
Removal of unnecessary elements and colors on the UI pages.
#### Description of Task to be completed?
Redesigning of the homepage.
Adding required form input on register page.
Adding party headquarters address on parties page.
Moving Express interest to main menu
Include mandate on express interest pop up.
Fixes on colors on admin dashboard
#### How should this be manually tested?
By visiting homepage, register page, parties page, admin dashboard.
#### What are the relevant pivotal tracker stories?
163537823

kenzdozz committed:
Initial commit

kenzdozz committed:
feat (register-page): created the register page
- create a registration page with form inputs: names, email address and password, national identification number, gender, accept terms
- create javascript validation function for forms
[Delivers #163310772]

kenzdozz committed:
feat(login-page): created the login page
- create a page with form inputs: email and password and a submit button
- created login.html and login.js
[Delivers #163310983]

kenzdozz committed:
feat(reset-password): create the reset password page
- create a page with form inputs: email address and submit button
- created reset-password.html and reset-password.js
[Delivers #163311097]

kenzdozz committed:
feat(admin-dashboard): Created the admin dashboard
- created an admin dashboard
[Delivers #163343240]

kenzdozz committed:
feat(manage-parties): created parties management page
- created form for party creation
- created view for all parties
- created dialog for deleting parties
[Dleivers #163311260]

kenzdozz committed:
Merge pull request #1 from kenzdozz/ft-register-page-163310772

feat (register-page): created the register page

kenzdozz committed:
Merge pull request #2 from kenzdozz/ft-login-page-163310983

feat(login-page): created the login page

kenzdozz committed:
feat(manage-office): created pages for offices management
- created a form with inputs: name, office type and submit button to create political offices
- create a view to display political offices
- created dialog to edit and delete an office
[Delivers #163311697]

kenzdozz committed:
feat(home-page): created the website homepage
- created the website homepage: index.html with js and css home.js and home.css
- linked other pages to the homepage
[Delivers #163348452]

kenzdozz committed:
feat(view-politicians): Created page to view politicians
- created a page to display all politicians
- created vote buttons to vote a politician for a political office
[Delivers #163311989]

kenzdozz committed:
feat(view-parties): created page to display parties
- created a page to display all political parties
[Delivers #163311981]

kenzdozz committed:
feat(express-interest): Implemented view for expressing interest
- created a modal with a form and inputs: political party and office
- fixed modal title on links on admin pages
[Delivers #163311904]

kenzdozz committed:
Merge pull request #3 from kenzdozz/ft-reset-password-163311097

feat(reset-password): create the reset password page

kenzdozz committed:
Merge pull request #4 from kenzdozz/ft-admin-dashboard-163343240

feat(admin-dashboard): Created the admin dashboard

kenzdozz committed:
Merge pull request #5 from kenzdozz/ft-manage-parties-163311260

feat(manage-parties): created parties management page

kenzdozz committed:
Merge pull request #6 from kenzdozz/ft-manage-office-163311697

feat(manage-office): created pages for offices management

kenzdozz committed:
Merge pull request #7 from kenzdozz/ft-home-page-163348452

feat(home-page): created the website homepage

kenzdozz committed:
Merge pull request #8 from kenzdozz/ft-view-politicians-163311989

feat(view-politicians): Created page to view politicians

kenzdozz committed:
Merge pull request #9 from kenzdozz/ft-view-parties-163311981

feat(view-parties): created page to display parties

kenzdozz committed:
Merge pull request #10 from kenzdozz/ft-express-interest-163311904

feat(express-interest): Implemented view for expressing interest

kenzdozz committed:
chore(gh-pages): Host UI on Github pages
- created gh-pages branch to host UI on Github pages
[Delivers #163376123]

kenzdozz committed:
chore(setup-project): Setup project, linting and testing
- initialize a node project
- install eslint
- install mocha and chai
[Delivers #163376698]

kenzdozz committed:
chore(unit-tests): Set up unit tests
- Wrote test for all API endpoints to manage political offices and parties
[Delivers #163403004]

kenzdozz committed:
feat(endpoint-create-party): Create API endpoint for party creation
- Set up express app
- create Party model, route and controller
- create validation middleware
[Delivers #163426328]

kenzdozz committed:
feat(endpoint-get-parties): API endpoint to Get all political parties
- created methods on Party model to return all parties
- created controller and route functions to return parties
[Delivers #163426987]

kenzdozz committed:
feat(ft-endpoint-get-specific-party): API endpoint to Get a specific political party
- created method on Party model to find a specific party
- created controller and route functions to return the specific party
[Delivers #163433964]

kenzdozz committed:
feat(ft-endpoint-edit-specific-party): API endpoint to Edit a specific political party
- created method on Party model to edit a specific party
- created controller and route functions to handle editing the specific party
[Delivers #163436818]

kenzdozz committed:
feat(delete-specific-party): API endpoint to Delete a particular party
- created method on Party model to remove a specific party
- created controller and route functions to delete the specific party
[Delivers #163439176]

kenzdozz committed:
feat(ft-endpoint-create-office): API endpoint for creation of political office
- created Office model, route and controller
- created controller and route functions to create the office
[Delivers #163466863]

kenzdozz committed:
feat(endpoint-get-offices): API endpoint to Get all political offices
- created methods on Party model to return all offices
- created controller and route functions to return offices
[Delivers #163466926]

kenzdozz committed:
feat(get-specific-office): API endpoint to Get a specific political office
- created method on Party model to find a specific political office
- created controller and route functions to return the specific political office
[Delivers #163467093]

kenzdozz committed:
Merge pull request #11 from kenzdozz/gh-pages

chore(gh-pages): Host UI on Github pages

kenzdozz committed:
Merge pull request #12 from kenzdozz/ch-setup-project-163376698

chore(setup-project): Setup project, linting and testing

kenzdozz committed:
Merge pull request #13 from kenzdozz/ft-endpoint-create-party-163426328

feat(endpoint create party): API endpoint for creation of party

kenzdozz committed:
Merge pull request #14 from kenzdozz/ft-endpoint-get-parties-163426987

feat(endpoint-get-parties): API endpoint to Get all political parties

kenzdozz committed:
Merge pull request #15 from kenzdozz/ft-endpoint-get-specific-party-163433964

feat(ft-endpoint-get-specific-party): API endpoint to Get a specific …

kenzdozz committed:
Merge pull request #16 from kenzdozz/ft-endpoint-edit-specific-party-163436818

feat(ft-endpoint-edit-specific-party): API endpoint to Edit a specifi…

kenzdozz committed:
Merge pull request #17 from kenzdozz/ft-endpoint-delete-specific-party-163439176

feat(delete-specific-party): API endpoint to Delete a particular party

kenzdozz committed:
Merge pull request #18 from kenzdozz/ft-endpoint-create-office-163466863

feat(ft-endpoint-create-office): API endpoint for creation of politic…

kenzdozz committed:
Merge pull request #19 from kenzdozz/ft-endpoint-get-offices-163466926

feat(endpoint-get-offices): API endpoint to Get all political offices

kenzdozz committed:
Merge pull request #20 from kenzdozz/ft-endpoint-get-specific-office-163467093

feat(get-specific-office): API endpoint to Get a specific political o…

kenzdozz committed:
chore(setup-travis=163470360): Added .travis.yml file

kenzdozz committed:
chore(setup-travis=163470360): Added travis badge to readme file

kenzdozz committed:
chore(setup-travis=163470360): Added travis badge to readme file

kenzdozz committed:
chore(setup-coveralls): Integrate test coverage reporting
- Integrate test coverage reporting from Coveralls with a badge in the ReadMe.
[Delivers #163470935]

kenzdozz committed:
chore(setup-coveralls): Integrate test coverage reporting
- Integrate test coverage reporting from Coveralls with a badge in the ReadMe.
[Delivers #163470935]

kenzdozz committed:
chore(setup-coveralls): Integrate test coverage reporting
- Integrate test coverage reporting from Coveralls with a badge in the ReadMe.
[Delivers #163470935]

kenzdozz committed:
chore(setup-coveralls): Integrate test coverage reporting
- Integrate test coverage reporting from Coveralls with a badge in the ReadMe.
- Improved on validateinputs middleware and utils helper on coverage
[Delivers #163470935]

kenzdozz committed:
chore(setup-coveralls): Integrate test coverage reporting
- Integrate test coverage reporting from Coveralls with a badge in the ReadMe.
- Included coveralls coverage badge in readme
[Delivers #163470935]

ascii-dev commented:
- Kindly change the PR to be raised against `gh-pages` instead of `develop`.
- Kindly make sure to link to your PT story under `What are the relevant pivotal tracker stories?`.

kenzdozz committed:
bug(ui-fixes): Some of the UI pages redesigned.
- Redesiged of the homepage.
- Added required form input on register page.
- Added party headquarters address on parties page.
- Moved Express interest to main menu.
- Included mandate on express interest pop up.
- Fixed colors on admin dashboard.
[Delivers #163537823]
